eq-author-api/migrations/dropDatatypeFieldFromPipedValues.js

```diff
@@ -0,0 +1,38 @@
+//This is an auto-generated file.  Do NOT modify the method signature.
+const cheerio = require("cheerio");
+const { flatMap } = require("lodash");
+
+module.exports = function dropDatatypeFieldFromPipedValues(questionnaire) {
+  const pages = flatMap(questionnaire.sections, section =>
+    flatMap(section.pages, page => page)
+  );
+
+  pages.map(page => {
+    ["title", "description", "guidance"].map(prop => {
+      let $elem;
+      if (prop === "title" && page.title) {
+        $elem = cheerio.load(page.title);
+      } else if (prop === "description" && page.description) {
+        $elem = cheerio.load(page.description);
+      } else if (prop === "guidance" && page.guidance) {
+        $elem = cheerio.load(page.guidance);
```

Comment:
You can get rid of this `if` statement with something like `$elem = cherio.load(page[prop]);`

Reply:
The script breaks if it tries to load an element that is empty.
Could I use something like this?
```js
      let $elem;
      if (page[prop]) {
        $elem = cheerio.load(page[prop]);
      } else {
        return;
      }
```